crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -107,10 +107,53 @@ impl CommandBase {
         Ok(self.client_config.get_mut().unwrap())
     }
 
+    pub fn spaces_api_auth(&self) -> Result<Option<APIAuth>, ConfigError> {
+        let repo_config = self.repo_config()?;
+        let team_id = repo_config.space_team_id();
+        let team_slug = repo_config.space_team_slug();
+
+        let token = self.user_config()?.token();
+
+        Ok(team_id.zip(token).map(|(team_id, token)| APIAuth {
+            team_id: team_id.to_string(),
+            token: token.to_string(),
+            team_slug: team_slug.map(|s| s.to_string()),
+        }))
+    }
+
+    pub fn api_auth(&self) -> Result<Option<APIAuth>, ConfigError> {
+        let repo_config = self.repo_config()?;
+        let team_id = repo_config.team_id();
+        let team_slug = repo_config.team_slug();
+
+        let token = self.user_config()?.token();
+
+        Ok(team_id.zip(token).map(|(team_id, token)| APIAuth {
+            team_id: team_id.to_string(),
+            token: token.to_string(),
+            team_slug: team_slug.map(|s| s.to_string()),
+        }))
+    }
```

Comment:
I'm confused as to why we have two identical functions.

Reply:
Yeah...me too. There was some debate about what to do around spaces auth vs regular API auth (cc @Zertsov). I don't recall the exact details but I remember Nathan saying something about just duplicating the auth and the API clients for now.

Reply:
Do we want them to be distinct types?

Reply:
Fixed now. Seems like we consolidated config